packages/@uppy/transloadit/src/index.ts

```diff
@@ -130,6 +130,12 @@ export interface TransloaditOptions<M extends Meta, B extends Body>
   waitForMetadata?: boolean
   importFromUploadURLs?: boolean
   alwaysRunAssembly?: boolean
+  /**
+   * Only use Transloadit for remote file uploads (such as from Google Drive).
+   * Enabling this means you have to install another plugin for local files,
+   * such as @uppy/aws-s3 or @uppy/xhr-upload.
+   */
+  companionOnly?: boolean
```

Comment:
I'm not sure if `companionOnly` is the right name for this. It makes it sound like we are only using Companion, but in fact we are also using Transloadit+assemblies. Maybe a better name would be something like `remoteOnly`? considering that what the option does is to make it only process remote files. (as opposed to processing both remote and local files without the option).

Reply:
Renamed to `onlyRemoteFiles`.